misk-redis/src/main/kotlin/misk/redis/RedisModule.kt

```diff
@@ -35,10 +37,19 @@ class RedisModule(
     )
 
     // Authenticate with the redis server
-    jedisPool.resource.use {
-      it.auth(replicationGroup.redis_auth_password)
+    try {
```

Comment:
this behavior is different now since you swallowed the exception and did not propagate

Reply:
Re-throwing the exception now